packages/identity/src/handle/index.ts

```diff
@@ -57,4 +59,39 @@ export class HandleResolver {
       return undefined
     }
   }
+
+  async resolveDnsBackup(handle: string): Promise<string | undefined> {
+    const backupIps = await this.getBackupNameserverIps()
+    if (!backupIps || backupIps.length < 1) return undefined
+    const resolver = new dns.Resolver()
+    resolver.setServers(backupIps)
+    let chunkedResults: string[][]
+    try {
+      chunkedResults = await resolver.resolveTxt(`${SUBDOMAIN}.${handle}`)
+    } catch (err) {
+      return undefined
+    }
+    return this.parseDnsResult(chunkedResults)
+  }
+
+  parseDnsResult(chunkedResults: string[][]): string | undefined {
+    const results = chunkedResults.map((chunks) => chunks.join(''))
+    const found = results.filter((i) => i.startsWith(PREFIX))
+    if (found.length !== 1) {
+      return undefined
+    }
+    return found[0].slice(PREFIX.length)
+  }
+
+  private async getBackupNameserverIps(): Promise<string[] | undefined> {
+    if (!this.backupNameservers) {
+      return undefined
+    } else if (!this.backupNameserverIps) {
+      const res = await Promise.all(
+        this.backupNameservers.map((h) => dns.lookup(h)),
+      )
```

Comment:
Could also do an `allSettled()` here, take the passing ones, and log the failing ones.  If you have a bad one I think that it just causes the dns lookups to occur again and `resolveDnsBackup()` to throw each time it's called.